maven.embedder/external/apache-maven-3.3.9-bin-notice.txt

```diff
@@ -0,0 +1,186 @@
+Apache Maven Distribution
+Copyright 2001-2015 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
```

Comment:
This part (line 4+5) is not needed (see: http://www.apache.org/dev/licensing-howto.html#bundle-asf-product)

Reply:
(Sorry for late reply.) Yes, that is not needed in the final NOTICE, and it won't be there - the task that merges notices strips there lines automatically. And it seems useful to keep manual edits to this file at minimum (this is actually concatenation of all NOTICEs from the whole Maven distro), to prevent mistakes.

Ok to merge?